templates/advantage/subscribe/form/_feature_coverage.html

```diff
@@ -3,8 +3,7 @@ <h3 class="p-stepped-list__title">Select your feature coverage</h3>
   <div class="row">
     <div class="col-12 subscribe-section">
       <p>
-        Essential features are already included in your subscription.<br />
-        You can add Infrastructure and Application features to your plan.
+        Essential features are already included in your subscription. <a href="/support">See what is included with Applications coverage.</a>
```

Comment:
I think you should leave the `<br />` in so the link wraps to a new line like the "What Ubuntu version are you running?" section above.

Reply:
agree